### ISSUE TITLE
[Bug]: cannot stop and restart database.

### ISSUE THREAD

**Issue by fengttt** [matrixorigin/matrixone#3340]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

start database, stop by ^C, restart, failed to restart.

{"level":"INFO","time":"2022/06/24 00:40:15.743227 -0700","caller":"txnbase/txnmgr.go:66","msg":"[INIT]","txnmgr":{"currId":0,"currTs":5030,"safeTs":5030}}
{"level":"INFO","time":"2022/06/24 00:40:15.743542 -0700","caller":"frontend/init_db.go:646","msg":"create table mo_global_variables failed.error:tae catalog: duplicate"}
{"level":"INFO","time":"2022/06/24 00:40:15.743642 -0700","caller":"db-server/main.go:367","msg":"Initialize catalog failed. error:tae catalog: duplicate"}

### Expected Behavior

Database should restart

**Comment by XuPeng-SH** [comment]
There is a config option "needInitdb" with a default value "true", which will init all system DB on every boot. You can set it to "false".
I think it is a temporary transition option and is not required now.

**Comment by fengttt** [comment]
Should this be default?   Please update default value.

**Comment by XuPeng-SH** [comment]
@daviszhen Could be set `needInitdb` default value to true? Or we don't need this config anymore.

**Comment by yingfeng** [comment]
Fix by #3709
